src/tezos/TezosConseilClient.ts

```diff
@@ -0,0 +1,69 @@
+import {ConseilOperator, ConseilQuery, ConseilSortDirection, ConseilServerInfo} from "../utils/v2/ConseilQuery";
+import {ConseilDataClient} from "../utils/v2/ConseilDataClient";
+import { url } from "inspector";
+
+/**
+ * Functions for querying the Conseil backend REST API v2
+ */
+export class TezosConseilClient extends ConseilDataClient {
+    BLOCKS = 'blocks';
+    ACCOUNTS = 'accounts';
+    OPERATION_GROUPS = 'operation_groups';
+    OPERATIONS = 'operations';
+
+    /**
+     * Returns a record set for a specific entity of the Tezos platform. Entity list and metadata can be retrieved using ConseilMetadataClient.
+     * 
+     * @param network Tezos network to query, mainnet, alphanet, etc.
+     * @param entity Entity to retrieve.
+     * @param filter Filter to apply.
+     */
+    async getTezosEntityData(serverInfo: ConseilServerInfo, network: string, entity: string, query: ConseilQuery): Promise<object> {
+        return super.executeEntityQuery(serverInfo, 'tezos', network, entity, query);
+    }
+
+    /**
+     * Get the head block from the Tezos platform given a network.
+     * 
+     * @param network Tezos network to query, mainnet, alphanet, etc.
+     */
+    async getBlockHead(serverInfo: ConseilServerInfo, network: string): Promise<object> {
+        const query = new ConseilQuery().setOrdering('level', ConseilSortDirection.DESC).setLimit(1);
+
+        return this.getTezosEntityData(serverInfo, network, this.BLOCKS, query);
+    }
+
+    async getBlock(serverInfo: ConseilServerInfo, network: string, hash: string): Promise<object> {
```

Comment:
missing meta-description, params, and descriptions for the rest of the functions (this one and ones below it)